udata/core/user/models.py

```diff
@@ -106,6 +109,11 @@ def organizations(self):
         from udata.core.organization.models import Organization
         return Organization.objects(members__user=self)
 
+    @cached_property
+    def involving_discussions(self):
```

Comment:
I'm not sure about this: it is used only once, the query can be inlined into `mark_as_deleted`